fivejars/openy_gated_content
# User segmentation

In this PR I've packed user segmentation logic:
Steps for testing:
1. [x] - Check that Virtual Y Video, Virtual Y blog post, Event Series has permission field after module install.
2. [x] - Open website as an anonymous user and enter virtual y using Dummy auth plugin.
4. [x] - Change permissions field at any of the virtual y content (uncheck Virtual Y, check Virtual Y Premuim).
5. [x] - Check that content has disappeared for dummy user (items that changed permissions from Virtual Y to Virtual Y Premium).
6. [x] - Check that Open Y administrator and Virtual Y editor has access to all Virtual content at admin pages and at Virtual Y
